examples/workflow.yaml

```diff
@@ -36,21 +36,30 @@ steps:
   - name: qubo
     type: qubo
 
-- name: get-exact-qubo-solution
+- name: get-qubo-solution-exact
   passed: [generate-random-qubo]
   config:
     runtime:
       language: python3
       imports: [z-quantum-core, z-quantum-qubo]
       parameters:
-        file: z-quantum-qubo/steps/qubo.py
-        function: get_exact_qubo_solution
+        file: z-quantum-qubo/steps/solvers.py
+        function: solve_qubo
   inputs:
   - qubo: ((generate-random-qubo.qubo))
     type: qubo
+  - solver_specs:
+      module_name: dimod
+      function_name: ExactSolver
+    type: specs
+  - sampler_params: null
+    type: specs
+
   outputs:
-  - name: exact_solution
+  - name: solution
```

Comment:
i still think exact_solution is a better name

Reply:
I have to stick to what the file saved in the step is – I removed the `get_exact_solution` step and now just use `ExactSolver` in a generic `solve_qubo` step (names might be off).
I hope it makes sense?